index.js

```diff
@@ -1,15 +1,27 @@
-/* jshint node: true */
-'use strict';
 
+'use strict';
+var path = require('path');
```

Comment:
Please stick with ember convention, remove 'use strict'; and the associated node module so that it is browser friendly.